src/cloud/architecture/starter-architecture.md

```diff
@@ -0,0 +1,135 @@
+We highly recommend testing every merchant and customer interaction in the Staging environment prior to deploying to the Production environment. See [Deploy your store][] and [Test deployment][].
+
+## Integration environment {#cloud-arch-int}
+
+Developers use the Integration environment to develop, deploy, and test:
+
+-  Magento application code
+-  Custom code
+-  Extensions
+-  Services
+
+You can have up to **two** active Integration environments. You create an Integration environment by creating a branch from the Staging branch. When you create an Integration environment, the environment name matches the branch name. An integration environment includes a web server and a database. It does not include all services, for example Fastly CDN and New Relic are not available.
+
+You can have an unlimited number of inactive branches for code storage. To access, view, and test an inactive branch, you must activate it.
+
+{% include cloud/note-enhanced-integration-envs-kb.md%}
+
+## Production and Staging technology stack {#technology}
+
+The Production and Staging environments include the following technologies. You can modify and configure these technologies through the [`.magento.app.yaml`][] file.
+
+-  Fastly for HTTP caching and CDN
+-  Nginx web server speaking to PHP-FPM, one instance with multiple workers
+-  Redis server
+-  Elasticsearch for searching for {{site.data.var.ece}} 2.2 and later
+
+### Services {#cloud-arch-services}
+
+{{site.data.var.ece}} currently supports the following services: PHP, MySQL (MariaDB), Elasticsearch (Magento 2.2.x and later), Redis, and RabbitMQ.
+
+Each service runs in a separate, secure container. Containers are managed together in the project. Some services are standard, such as the following:
+
+-  HTTP router (handling incoming requests, but also caching and redirects)
+-  PHP application server
+-  Git
+-  Secure Shell (SSH)
+
+You can have multiple applications running in the same project. Building a microservice-oriented architecture with {{site.data.var.ee}} is as easy as managing a monolithic application.
+
+### Software versions {#cloud-arch-software}
+
+{{site.data.var.ece}} uses the Debian GNU/Linux operating system and the [NGINX](https://glossary.magento.com/nginx) web server. You cannot upgrade this software, but you can configure versions for the following:
+
+-  [PHP][]
+-  [MySQL][]
+-  [Redis][]
+-  [RabbitMQ][]
+-  [Elasticsearch][]
+
+In the Staging and Production environments, you use Fastly for CDN and caching. When your environment is initially provisioned, we install the latest version of the Fastly CDN extension for Magento. You can upgrade the extension to get the latest bug fixes and improvements. See [Fastly CDN module for Magento 2][]. You also have access to [New Relic][] for performance management.
+
+You use the following files to configure the software versions that you want to use in your implementation.
+
+-  [`.magento.app.yaml`][magento.app.yaml]
```

Comment:
This link works just fine. So I suggested this for the broken one above. If you change the link definition, then you'd have to change both links. But the suggestion above should be okay.